packages/rln/src/rln.ts

```diff
@@ -25,26 +26,22 @@ import type {
 import { KeystoreEntity, Password } from "./keystore/types.js";
 import verificationKey from "./resources/verification_key";
 import * as wc from "./resources/witness_calculator";
+import { WitnessCalculator } from "./resources/witness_calculator";
 import { extractMetaMaskSigner } from "./utils/index.js";
 import { Zerokit } from "./zerokit.js";
 
 const log = new Logger("waku:rln");
 
-async function loadWitnessCalculator(): Promise<wc.WitnessCalculator> {
-  const res = await fetch("/base/rln.wasm");
-  if (!res.ok) {
-    throw new Error(`Failed to fetch rln.wasm: ${res.statusText}`);
-  }
-  const witnessBuffer = await res.arrayBuffer();
-  return wc.builder(new Uint8Array(witnessBuffer), false);
+async function loadWitnessCalculator(): Promise<WitnessCalculator> {
+  const url = new URL("./resources/rln.wasm", import.meta.url);
+  const response = await fetch(url);
```

Comment:
there is no error handling anymore, why?

Reply:
same for `loadZkey`

Reply:
added back 👍